modules/home/Number.tsx

```diff
@@ -0,0 +1,39 @@
+'use client';
+import React, { useEffect, useRef } from 'react';
+import { animate, inView } from 'motion';
+
+const counter = (elementRef: HTMLDivElement, start: number, end: number) => {
+  let current = start;
+  let range = end - start;
+  const duration = Math.sqrt(Math.log(range));
+  animate(
+    (progress: number) => {
+      current = Math.floor(start + progress * range);
+      elementRef.textContent = `${current}`;
```

Comment:
the `current` variable is not really needed anymore, but that's a nitpick
```suggestion
      elementRef.textContent = `${Math.floor(start + progress * range)}`;
```